test/api/unit/models/build.test.js

```diff
@@ -52,16 +52,44 @@ describe('Build model', () => {
   });
 
   describe('after create hook', () => {
+    it('should send a build new build message', (done) => {
+      const startedAt = new Date();
+      factory.build({ state: 'processing', startedAt })
+        .then((build) => {
+          expect(build.completedAt).to.be.null;
+          return build.updateJobStatus({ status: 'success' });
+        })
+        .then((build) => {
+          const queuedBuild = sendMessageStub.getCall(0).args[0];
+          const buildCount = sendMessageStub.getCall(0).args[1];
+
+          expect(sendMessageStub.called).to.be.true;
+          expect(queuedBuild.id).to.equal(build.id);
+          expect(buildCount).to.equal(1);
+          expect(build.state).to.eql('success');
+          expect(build.completedAt).to.be.a('date');
+          expect(build.startedAt).to.eql(startedAt);
+
+          done();
+        })
+        .catch(done);
+    });
+
     it('should send a build new build message', (done) => {
       factory.build()
-        .then(build => build.updateJobStatus({ status: 'complete' }))
+        .then((build) => {
```

Comment:
it looks like these two specs have the same name? It also looks like here we're testing `updateJobStatus` again, maybe tests for that should be elsewhere?